app/frontend/users/UserSessionResetPassword.res

```diff
@@ -109,6 +147,8 @@ let renderUpdatePassword = (
 @react.component
 let make = (~token, ~authenticityToken) => {
   let (newPassword, setNewPassword) = React.useState(() => "")
+  let zxcvbnRes = zxcvbn(newPassword, None)
```

Comment:
1. We should _really_ try to pass in the user's name and email components over to zxcvbn at this point. The user has been authenticated, so it's not like this information isn't retrievable at this point - just that it isn't _readily available_ unlike the _user edit_ form.
2. zxcvbn also strongly suggests that we include the website name into this array. Passing in `current_school.name` somehow into this array is also probably a really good idea.